packages/tests/src/node/dockerode.ts

```diff
@@ -159,18 +157,19 @@ export default class Dockerode {
   }
 
   async stop(): Promise<void> {
-    if (!this.container) throw "containerId not set";
-
-    log(
-      `Shutting down container ID ${
-        this.containerId
-      } at ${new Date().toLocaleTimeString()}`
-    );
+    if (!this.container) {
+      log("ContainerId not set");
+    } else {
+      log(
+        `Shutting down container ID ${
+          this.containerId
+        } at ${new Date().toLocaleTimeString()}`
+      );
 
-    await this.container.stop();
-    await this.container.remove();
+      await this.container.stop();
```

Comment:
With this line the tests are failing with:
Error: (HTTP code 404) no such container - No such container: 07766efa1ac57a0140a5f91cfac6694505094f837bf2c993b623353ce9f731e4 

In the past this code was not executed because we were calling: 

await this.docker?.container?.stop();

and I just fixed it now to:

await this.docker?.stop();